src/robot-arm-element.ts

```diff
@@ -0,0 +1,106 @@
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ElementPin } from '.';
+
+@customElement('wokwi-robot-arm')
+export class RobotArmElement extends LitElement {
+  /**
+   * Angle of shoulder
+   */
+  @property() sholderAngle = 297;
+
+  /**
+   * Angle of elbow
+   */
+  @property() elbowAngle = 52;
+
+  /**
+   * Angle of wrist
+   */
+  @property() wristAngle = 56;
+
+  /**
+   * Angle of gripper
+   */
+  @property() gripperAngle = 270;
+
+  /**
+   * The color of the gripper
+   */
+  @property() gripperColor = '#eeeedd';
+
+  /**
+   * The color of the arm
+   */
+  @property() armColor = '#ffffdd';
+
+  /**
+   * The color of the joints
+   */
+  @property() jointColor = '#111111';
+
+  readonly pinInfo: ElementPin[] = [
+    { name: 'PWM1', y: 195, x: 25, number: 1, signals: [{ type: 'pwm' }] },
+    { name: 'PWM2', y: 195, x: 30, number: 2, signals: [{ type: 'pwm' }] },
+    { name: 'PWM3', y: 195, x: 35, number: 3, signals: [{ type: 'pwm' }] },
+    { name: 'PWM4', y: 195, x: 40, number: 3, signals: [{ type: 'pwm' }] },
+  ];
+
+  render() {
+    /* eslint-disable prettier/prettier */
```

Comment:
why?

Reply:
When I let VS code format the SVG it becomes un-readable and I hate the warnings. But if you really want it to wrap all those lines I can remove it.

Reply:
Yes, prettier SVG formatting isn't that great.

But when the choice is between not-so-pleasing but consistent SVG formatting or getting the indentation wrong by the next developer who works on this code, I prefer the former.

Also, prettier should automatically format the code whenever you commit anyway. Didn't it?